aten/src/ATen/native/PointwiseOps.cpp

```diff
@@ -38,7 +38,7 @@ Tensor& addcmul_out(
     Scalar value) {
   checkBackend("addcmul_cpu", result, self.type().backend());
   auto iter = at::TensorIterator();
-  iter.check_and_add_output(result);
+  iter.add_output(result);
```

Comment:
Shouldn't `iter.set_check_overlap(true)` be here to preserve the old behavior?

Reply:
addressed